fe/fe-core/src/main/java/org/apache/doris/common/util/DynamicPartitionUtil.java

```diff
@@ -348,13 +363,32 @@ public static void registerOrRemoveDynamicPartitionTable(long dbId, OlapTable ol
             analyzedProperties.put(DynamicPartitionProperty.CREATE_HISTORY_PARTITION, val);
         }
 
+        int historyPartitionNum = 0;
```

Comment:
It is better to keep the same style as the existing code. 
You can refer to `REPLICATION_NUM` section.